ci/master/variables.yaml

```
@@ -19,7 +19,7 @@ projects:
 
   - name: q2cli
     deps: [qiime2]
-    test_runner: nosetests q2cli
+    test_runner: py.test q2cli
```

We need to include the `--pyargs` flag so that py.test will test the built package, rather than the source.

```suggestion
    test_runner: py.test --pyargs q2cli
```